services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/audit/KinesisAuditRecord.java

```diff
@@ -0,0 +1,64 @@
+package org.sagebionetworks.repo.manager.audit;
+
+import org.sagebionetworks.kinesis.AwsKinesisLogRecord;
+
+public class KinesisAuditRecord implements AwsKinesisLogRecord {
+
+    public static final String STREAM_NAME = "auditData";
+
+    private Long timestamp;
+    private Object payload;
```

Comment:
Object->JSONEntity

Reply:
Add an abstract AwsKinesisLogRecord that handles  stack, instance.  Also Marco suggests adding a generic AwsKinesisLogRecord<? extends JSONEntity>.  
AwsKinesisLogRecord<AccessRecord>